packages/p2p/src/components/advertiser-page/block-user/block-user-empty.scss

```diff
@@ -0,0 +1,14 @@
+.block-user-empty {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    &__icon {
+        margin-top: 2.2rem;
+    }
+
+    &__text {
+        margin: 3rem 0 0.8rem;
+        font-size: 1.606rem;
```

Comment:
Set font size to conform design requirements of `243x24`:
![image](https://user-images.githubusercontent.com/103016120/174774472-4112d10b-10f0-4cd0-96a9-88dd8eae48ae.png)

Reply:
Cuz QA says it should be `243x24` so I had to adjust the font size manually 😢